06-aks-cluster.md

```diff
@@ -1,12 +1,12 @@
 # Deploy the AKS Cluster
 
-Now that the your [ACR instance is deployed and ready to support cluster bootstrapping](./05-bootstrap-prep.md), the next step in the [AKS Baseline reference implementation](./) is deploying the AKS cluster and its remaining adjacent Azure resources.
+Now that the your [ACR instance is deployed and ready to support cluster bootstrapping](./05-bootstrap-prep.md), the next step in the [AKS baseline reference implementation](./) is deploying the AKS cluster and its remaining adjacent Azure resources.
```

Comment:
"Now that your ACR"

Reply:
Thanks, fixed in https://github.com/mspnp/aks-baseline/pull/274/commits/d7ee7e870ccf07091f725484fef2bc57b7d39be0